### PR TITLE
new site and copyright

### PR DESCRIPTION
**Pull request by MaksimChepelev**

### CODE REVIEW

File: client/pom.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2016, 2017 Iotracks, Inc.
+    Copyright (c) 2018 Edgeworx, Inc.
```
Comment (ekrylovich) [inline client/pom.xml]: ioFog????

Comment (ekrylovich) [reply client/pom.xml]: ANd really should we change year here?

File: docs/ioFabric-Architecture.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ A list of the modules in the ioFabric application can be found immediately below
...
 
-At iotracks, we believe that one of the next great challenges in computing technology is handling the rapidly increasing number of data sources, their widening variety, and the need to add and remove them dynamically without rebuilding solution code. Along with these voluminous data sources comes massive amounts of actual data. We believe that processing should be moved to where the data is, instead of moving large amounts of data all the way to a central backend for processing.
+At iofog, we believe that one of the next great challenges in computing technology is handling the rapidly increasing number of data sources, their widening variety, and the need to add and remove them dynamically without rebuilding solution code. Along with these voluminous data sources comes massive amounts of actual data. We believe that processing should be moved to where the data is, instead of moving large amounts of data all the way to a central backend for processing.
```
Comment (epankou) [inline docs/ioFabric-Architecture.md]: I think here should be edgeworx, not iofog

File: docs/ioFabric-Command-Line-Specification.md [hunk 3]
```diff
@@ -229,7 +229,7 @@ iofabric info
 Instance ID               : sdfh43t9EFHSD98hwefiuwefkshd890she
 IP Address                : 201.43.0.88
 Network Adapter           : eth0
-ioFabric Controller       : http://iotracks.com/controllers/2398yef
+ioFabric Controller       : http://iofog.org/controllers/2398yef
```
Comment (epankou) [inline docs/ioFabric-Command-Line-Specification.md]: should we change old iofabric name?

Comment (ekrylovich) [reply docs/ioFabric-Command-Line-Specification.md]: +1

File: docs/ioFabric-Architecture.md (hunk 2 above)
Comment (ekrylovich) [reply docs/ioFabric-Architecture.md]: I think nothing in open source product should link to commercial Edgeworx

Comment (epankou) [reply docs/ioFabric-Architecture.md]: ok then what does 'iofog' mean here? 'iotracks' earlier was supposed to mean "iotracks team"

File: client/pom.xml (hunk 1 above)
Comment (MaksimChepelev) [reply client/pom.xml]: 2018 Edgeworx. I got this copyright from Kilton

File: daemon/src/org/eclipse/iofog/command_line/CommandLineAction.java [hunk 4]
```diff
@@ -293,7 +293,7 @@ private static String showHelp() {
                 "										  shut down\\n" +
                 "\\n" +
                 "\\n" +
-                "Report bugs to: bugs@iotracks.com\\n" +
+                "Report bugs to: bugs@iofog.org\\n" +
                 "ioFog home page: http://iofog.com\\n" +
```
Comment (ilaryionava) [inline daemon/src/org/eclipse/iofog/command_line/CommandLineAction.java]: iofog.org ?

File: daemon/src/org/eclipse/iofog/utils/logging/LoggingService.java [hunk 5]
```diff
@@ -97,7 +97,7 @@ public static void setupLogger() throws IOException {
 	
 		logFileHandler.setFormatter(new LogFormatter());
 	
-		logger = Logger.getLogger("com.iotracks.iofog");
+		logger = Logger.getLogger("org.iofog");
```
Comment (ilaryionava) [inline daemon/src/org/eclipse/iofog/utils/logging/LoggingService.java]: org.eclipse.iofog ?

File: docs/ioFabric-Architecture.md [hunk 6]
```diff
@@ -126,7 +126,7 @@ Other than serving as the status repository, the only activity that the Status R
...
+The Process Manager module is in charge of starting, stopping, and generally controlling the processes that run in ioFabric. In the case of this particular ioFabric version, the processes take the form of Linux kernel containers running via Docker. These processes are often called ioElement containers or sometimes just elements in the overall iofog system. They are the actual computing tasks that are taking place on the iofog I/O compute fabric. There is no need for ioFabric to have any awareness of what the processes might be. It only needs to manage them properly and manage them all exactly the same. Through that standardization, all ioElement containers become portable and re-usable from one part of the fabric to another.
```
Comment (ilaryionava) [inline docs/ioFabric-Architecture.md]: ioFabric ?

File: docs/ioFabric-Architecture.md [hunk 7]
```diff
@@ -126,7 +126,7 @@ Other than serving as the status repository, the only activity that the Status R
...
 
 The Process Manager needs to interface with the Docker daemon to get a lot of its work done. It does that through the socket defined in the ioFabric configuration. The default is for Docker to communicate using Unix sockets, which is the most secure and is very fast. Therefore the default configuration in ioFabric is to use that default Docker setup. If the ioFabric user wants to run Docker over a TCP/IP socket, they are allowed to do so. As long as they enter the correct socket setting in the ioFabric configuration, everything should work as expected.
```
Comment (ilaryionava) [inline docs/ioFabric-Architecture.md]: iofabric ?

File: docs/ioFabric-Fabric-Controller-API-Specification.md [hunk 8]
```diff
@@ -505,7 +505,7 @@ This endpoint provides the list of Docker container registries that the ioFabric
                 	"requirescert":false,
                 	"username":"iofabric",
                 	"password":"abc123",
-                	"useremail":"iofabric_user@iotracks.com"
+                	"useremail":"iofabric_user@iofog.org"
```
Comment (ilaryionava) [inline docs/ioFabric-Fabric-Controller-API-Specification.md]: iofabric_user ??????

File: docs/ioFabric-Packaging-Instructions.md [hunk 9]
```diff
@@ -58,7 +58,7 @@ Change directories into the Debian packaging folder. Make a new package build, a
 
 	Publish the package file to the Package Cloud with the new file name:
 
-	package_cloud push iotracks/iofabric XXXXXXXX.deb
+	package_cloud push iofog/iofabric XXXXXXXX.deb
```
Comment (ilaryionava) [inline docs/ioFabric-Packaging-Instructions.md]: iofog/iofabric ????

File: docs/ioFabric-System-Container-Requirements.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 
 Every ioFabric instance comes with some default containers. These "system containers" provide functionality that enhances the ioFabric software. The reason this functionality comes in the form of a container is so it can be updated easily for all ioFabric instances. One of the best things about ioFabric is that once the base software that handles containers is running, everything else can be implemented as a container and this minimizes the versioning problems that happen with distributed software.
```
Comment (ilaryionava) [inline docs/ioFabric-System-Container-Requirements.md]: iofabric ?

**Comment by ilaryionava** [comment]
all references of iotracks & iofabric, fabric, etc. should be replaced

**Pull request merged**